plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -251,6 +268,10 @@ func (h *HTTPListenerV2) serveWrite(res http.ResponseWriter, req *http.Request)
 			}
 		}
 
+		if h.PathTag != "" {
+			m.AddTag(h.PathTag, req.URL.Path)
+		}
```

Comment:
With `PathTag` being a `bool`:
```suggestion
		if h.PathTag {
			m.AddTag("path", req.URL.Path)
		}
```